scripts/variantstore/TERRA_QUICKSTART.md

```diff
@@ -21,38 +21,53 @@ This quickstart assumes that you are familiar with Terra workspaces, the data mo
+## 2. Reblock samples
+Run the GvsAoUReblockGvcf workflow on the samples
+
+These are the required parameters which must be supplied to the workflow:
+
+| Parameter      | Description |
+| ----------------- | ----------- |
+| gvcf | The gvcf file to reblock |
+| ref_dict      		| The reference dictionary to use       |
+| ref_fasta      	| The reference to use      |
+| ref\_fasta_index   | The reference index       |
 
-## 1.1 Assign Gvs IDs
-To optimize the internal queries, each sample must have a unique and consecutive integer ID assigned. Run the `GvsAssignIds` workflow, which will create an appropriate ID for each sample in the sample set and update the BigQuery dataset with the sample name to ID mapping info.
+
+## 3. Load data
+### 3.1 Assign Gvs IDs (suggest renaming to GvsPrepareSamples)
+To optimize the internal queries, each sample must have a unique and consecutive integer ID assigned. Run the `GvsAssignIds` workflow on a sample set, which will create an unique gvs id, update the BQ sample\_info table (creating it if it doesn't exists), and update the data model. This workflow takes care of creating the BQ vet and ref_ranges tables needed for the sample ids generated.
 
 These are the required parameters which must be supplied to the workflow:
 
 | Parameter      | Description |
 | ----------------- | ----------- |
 | project_id | The name of the google project containing the dataset |
 | dataset_name      | The name of the dataset you created above       |
+| external\_sample_names      | Sample ids from the data model that will be mapped to gvs ids       |
+| workspace_namespace      | The namespace of the workspace       |
+| workspace_name      | The name of the workspace       |
 
 
-## 1.2 Load data
+### 3.2 Import Samples
 
-Next, your re-blocked gVCF files should be imported into GVS by running the `GvsImportGenomes` workflow.
+Next, your re-blocked gVCF files should be imported into GVS by running the `GvsImportSample` workflow which now works on individual samples. The workflow will check whether data for that sample has already been loaded into GVS.
 
 These are the required parameters which must be supplied to the workflow:
 
 | Parameter      | Description |
 | ----------------- | ----------- |
 | project_id | The name of the google project containing the dataset |
 | dataset_name      | The name of the dataset you created above       |
-| output_directory | A unique GCS path to be used for loading, can be in the workspace bucket.  E.g. `gs://fc-124-12-132-123-31/gvs/demo1`)
-
+| external\_sample_name | The name of the sample from the data model |
+| gvs\_sample_id | The gvs_id from the data model |
```

Comment:
Can we just grab this from BigQuery?   One less thing for the user to do/mess up :D

Reply:
Since we are now running ingest per sample, I didn't want each vm to have to make a query to the sample table for the id. Any ideas of other ways were we won't run into query limits?